app/uk/gov/hmrc/customs/declaration/xml/MdgPayloadDecorator.scala

```diff
@@ -18,14 +18,14 @@ package uk.gov.hmrc.customs.declaration.xml
 
 import org.joda.time.DateTime
 import org.joda.time.format.ISODateTimeFormat
-import uk.gov.hmrc.customs.declaration.model.Csp
+import uk.gov.hmrc.customs.declaration.model.{Csp, FieldsId, SubscriptionFieldsId}
```

Comment:
unused import?  Also in Customs-inventory-linking-exports we use SubscriptionFieldsId...Not for this review but we might want to record somewhere that we have different naming in different services. Should be consistent across them